Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -1463,6 +1463,29 @@ static void drawPDFPage(PDFDocument *pdfDocument, CFIndex pageIndex, CGContextRe
         manager->processSystemDidWake();
 }
 
+NSObject *WebPage::accessibilityObjectForMainFramePlugin()
+{
+    if (!m_page)
+        return nil;
+
+#if ENABLE(PDF_PLUGIN)
+    if (RefPtr pluginView = mainFramePlugIn())
+        return pluginView->accessibilityObject();
+#endif
```

Comment:
Should the `if (!m_page)` check be moved into the `if ENABLE(PDF_PLUGIN)` block, since `!if ENABLE(PDF_PLUGIN)` always returns nil anyways?

Reply:
Sure